src/chttpd_cors.erl

```diff
@@ -176,15 +180,15 @@ headers(Req, RequestHeaders, Origin, Config) ->
         true ->
             AcceptedOrigins = get_accepted_origins(Req, Config),
             CorsHeaders = handle_headers(Config, Origin, AcceptedOrigins),
-            maybe_apply_headers(CorsHeaders, RequestHeaders);
+            maybe_apply_headers(Config, CorsHeaders, RequestHeaders);
```

Comment:
Personally I'd get `exposed_headers` out of config on a line above to make input parameters in `maybe_apply_headers` consistent to be all headers, but this is a question of a personal taste, so feel free to ignore.